Tools/PowershellModule/doc/new-private-repository-azure.md

```diff
@@ -1,238 +1,226 @@
+The Windows Package Manager rest source provides a location for hosting your Application Manifests. The `Microsoft.WinGet.Source` PowerShell module provides the [Remove-WinGetManifest](.\PowerShell\Remove-WinGetManifest.md) cmdlet that will remove a specific Application Manifest from the specified Windows Package Manager rest source.
 
-> [!Note]
-> The PowerShell Module must be re-imported each time the PowerShell window is closed.
+The `Remove-WinGetManifest` PowerShell cmdlet supports targeting an existing Windows Package Manager REST source for a specific Manifest Identifier. For more information on how to use this cmdlet, use the `Get-Help Remove-WinGetManifest -Full` or visit the [Remove-WinGetManifest article in the PowerShell docs](.\PowerShell\Remove-WinGetManifest.md).
 
-**How to:**
+The PowerShell Module must be re-imported each time the PowerShell window is closed. To remove an Application Manifest open the Administrative PowerShell Window and run the following:
 
-1. From the Administrative PowerShell Window run the following:
 ```PowerShell
 PS C:\> Remove-WinGetManifest -FunctionName "contoso" -ManifestIdentifier "Windows.PowerToys"
 ```
 
-## Manually create a REST source
+## Manage Windows Package Manager rest source manually
 
 The following instructions assumes the following Azure objects are named as follows:
+
 | Azure Resource          | Value                      |
 |-------------------------|----------------------------|
-| Azure Resource Group    | WinGet_PrivateRepo         |
+| Azure Resource Group    | WinGet_restsource          |
 | Azure Subscription Name | Contoso Azure Subscription |
 | Azure Location          | West US                    |
 | Azure Key Vault Name    | contoso-keyvault-demo      |
 
 *We assume that the Resource Group and Subscription are pre-existing in your Azure Tenant.*
 
-### Extract the Windows Package Manager REST Source
+### Extract the Windows Package Manager Rest Source
 
-The Windows Package Manager REST Source contains the APIs required to provide a Windows Package Manager REST source.
+The Windows Package Manager REST Source contains the APIs required to provide a Windows Package Manager REST source. To download a local copy of the Windows Package Manager REST Source from GitHub:
 
-**How to:**
-
-1. Download a local copy of the Windows Package Manager REST Source from GitHub (github: [winget-cli-restsource](https://github.com/microsoft/winget-cli-restsource/releases))
-    1. Download *.Zip file from the latest release.
-    1. Extract the newly downloaded ZIP file to *C:\Projects\\*
+1. Visit [winget-cli-restsource](https://github.com/microsoft/winget-cli-restsource).
```

Comment:
Directions should point to releases, not source code.

Reply:
That's old.